src/pages/Details/VideoDetails.page.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useContext, useEffect } from 'react';
+import { useParams } from 'react-router';
+import VideoRelated from '../../components/RelatedVideos';
+import VideoDetail from '../../components/VideoDetail';
+import { VideosContext } from '../../context/videoContext';
+import {
+  MainContainer
+} from './VideoDetails.styled';
+
+const VideoDetails = () => {
+  const videoContext = useContext(VideosContext);
+  const { getVideoDetails } = videoContext;
+
+  const { id } = useParams();
+
+  useEffect(() => {
+    getVideoDetails(id);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [id]);
```

Comment:
The eslint rule react-hooks/exhaustive-deps exists for one reason, it can alert us for possible bugs or unexpected behaviors on the app.

Please remove this ignore rule comment and try to fix the eslint message.